docs/front-end/frameworks-used.md

```diff
@@ -38,6 +38,21 @@ The following variables are required by the front end and should be setup as Git
 - `SENTRY_ORG`, the organisation specified in Sentry
 - `SENTRY_AUTH_TOKEN`, authentication token used for all communication with Sentry
 
+## Feature Flags
+
+The following feature flag is set in the next root .env file: 
+- `RELEASE_VERSION`
+
+This feature flag can be set to 
+- 'mvp' or
+- 'full'.
+
+Any other value, including the absense of this variable will result in the application falling back on the 'full' version.
```

Comment:
I think that we might need a `,` after the word `absense`. And also I think it should be `absence`.